app/views/shared/_comment_block.html.erb

```diff
@@ -0,0 +1,48 @@
+<div class="comment-bucket" id='comment-<%= "#{comment.id}" %>' >
+
+  <table>
+    <tr>
+      <td>
+      <% if current_user.present? %>
+        <% voteLookup = HackerNews::VoteLookupService.new(user: current_user, votable: comment) %>
+        <% if voteLookup.vote.present? %>
+          <%= link_to vote_path(voteLookup.vote.id), method: :delete, class: 'down-vote-link' do %>
+            <span class="glyphicon glyphicon-chevron-up"></span>
+          <% end %>
+        <% else %>
+          <%= link_to votes_path(votable_id: comment.id, votable_type: comment.class.to_s), method: :post, class: 'up-vote-link' do %>
+            <span class="glyphicon glyphicon-chevron-up"></span>
+          <% end %>
+        <% end %>
+      <% else %>
+        <span class="glyphicon glyphicon-chevron-up" id="false-link"></span>
+      <% end %>
+     </td>
+     <th class="comment-username">
+       <span><%= "#{comment.user.username}" %></span>
```

Comment:
If `User#to_s` points to the `username` method, you can just do `"#{comment.user}"`, but in this case, you aren't combining any strings so you probably can ditch the interpolation altogether:
```
<span><%= comment.user.username %></span>
```